src/components/timeline/TimelineHistogram.svelte

```diff
@@ -365,10 +402,17 @@
       constraintViolationsToRender.length ? histogramHeight * 1.5 : histogramHeight * 2
     }px; border-bottom: ${constraintViolationsToRender.length < 1 ? '1px solid var(--st-gray-20)' : 'none'}`}
   >
-    {#each activityHistValues as bin, index (index)}
-      <div class="bin" style={`height: ${(bin / activityHistMax) * 100}%;`} />
+    {#each aggregateHistogram as binPair, index (index)}
```

Comment:
A little confused, does this mean that events will only be shown in the histogram if there is no activity for the bin at index N?

Reply:
Looks like it - we should chat about behavior for the timeline histogram and/or punt on showing events in here until we settle on a design.
![image](https://github.com/user-attachments/assets/dfef9bab-6a14-4d3c-b9d4-537d5bf90e79)

Reply:
Solution - draw the taller of the two bars in case of overlap (or consider stacking?).